test/cli/iGenVar_cli_test.cpp

```diff
@@ -702,6 +706,7 @@ TEST_F(iGenVar_cli_test, dataset_single_end_mini_example)
     std::string output_err_str((std::istreambuf_iterator<char>(output_err_file)),
                                 std::istreambuf_iterator<char>());
     EXPECT_EQ(result.err, output_err_str);
+
```

Comment:
I guess this change was unintentional

Reply:
You removed the wrong empty line (at the end of the file), but I don't mind much ;)